django/contrib/admin/templates/admin/app_list.html

```diff
@@ -8,29 +8,33 @@
           <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
         </caption>
         {% for model in app.models %}
-          <tr class="model-{{ model.object_name|lower }}{% if model.admin_url in request.path|urlencode %} current-model{% endif %}">
-            {% if model.admin_url %}
-              <th scope="row"><a href="{{ model.admin_url }}"{% if model.admin_url in request.path|urlencode %} aria-current="page"{% endif %}>{{ model.name }}</a></th>
-            {% else %}
-              <th scope="row">{{ model.name }}</th>
-            {% endif %}
+          {% with model_name=model.object_name|lower %}
```

Comment:
As far as I'm aware it's the same as `model.model_name`, so with is unnecessary.

Reply:
I suggested that to maintain the existing definition in the `tr` row (otherwise we would be changing semantics from what's in `main`). Do you think it's safe to remove lower?

Reply:
I didn't suggest to remove `lower`. I suggested to use `model_name` instead of `object_name.lower`:

https://github.com/django/django/blob/0eaad0bb593512bedbc96e52bdcf03213d691ab3/django/db/models/options.py#L178-L179

Reply:
Nvd, we have here `model_dict`, not `model._meta` so `model_name` is not available.